libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/NavigationRouteProcessor.java

```diff
@@ -27,7 +27,7 @@
 class NavigationRouteProcessor {
 
   private static final int ONE_INDEX = 1;
-  private static final long ONE_SECOND_IN_MILLISECONDS = 1000L;
+  private static final double ONE_SECOND_IN_MILLISECONDS = 1000.0;
```

Comment:
using double here so we dont truncate below. not sure if we really need subsecond accuracy, but its there if we need :smile: